### DISCUSSION TITLE
Why is the order of extracting the contents in the table cells wrong?

### DISCUSSION THREAD

**Discussion by xielulu1994** [jsvine/pdfplumber#1069] [Ask for help with specific PDFs]
## Describe the bug
extract_table() to extract table content, and find that the order of extracted text in individual cells is inconsistent with the original text.

![image](https://github.com/jsvine/pdfplumber/assets/42051898/ec802cea-7d54-487f-9560-16cce57e9d6d)

pdf table：
![image](https://github.com/jsvine/pdfplumber/assets/42051898/67b30a7b-92c2-483c-9389-3d1c5ab2669f)

## Code to reproduce the problem
```python
table_text_items: List[tuple] = []
    with pdfplumber.open(file_path) as pdf:
        for page in pdf.pages:
            table = page.extract_table()
            lines: List[str] = []
            if table:
                for row in table:
                    for line in [item for item in row if item is not None]:
                        if line:
                            lines.extend(line.split("\n"))
            if lines:
                table_text_items.append((page.page_number, lines))
```

## PDF file

*Please attach any PDFs necessary to reproduce the problem.*

*If you need to redact text in a sensitive PDF, you can run it through [JoshData/pdf-redactor](https://github.com/JoshData/pdf-redactor).*

**Comment by jsvine** [comment]
Hi @xielulu1994, and thanks for your interest in `pdfplumber`. Without access to the PDF itself, it is difficult to provide tested advice. However, from looking at the screenshots, my guess is that the bounding boxes of the characters in the `cd \d`[...] text are a bit above the rest of the text. Try this:

```python
page.extract_table({ "text_y_tolerance": 5 })
```

... although you may have to try adjusting that `5` value higher or lower, depending on the specifics of the PDF.
